### ISSUE TITLE
BUG - Downloading of Backup becomes a document and not a zip file

### ISSUE THREAD

**Issue by ErnestPH** [AzuraCast/AzuraCast#2141]
Server is uBuntu 18 with Azuracast v0.9.7.1.

After a successful backup. When downloading the zip file it is downloaded as
v0.9.7.1_yukon_2020.

Should be v0.9.7.1_yukon_2020.zip

Using Safari browser to download it

**Comment by ErnestPH** [comment]
FYI - Google Chrome Mac is working ok. Only Safari. I guess Vaalyn can help this legit bug.

**Comment by SlvrEagle23** [comment]
@Vaalyn Following up on this..what's the solution here for Safari users? We're sending the correct "attachment" response type with the correct filename, so it just seems to be disregarding it for some reason.

**Comment by Vaalyn** [comment]
Sorry for the delay.

I've looked at this issue, can replicate it and also fixed it in the PR that I just submitted.

The gist is that some browsers seem to prefer the [`download` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) with the name as the value. I'm not sure if Safari completely ignores the `content-disposition` header or if there is a trick to it.

I currently don't have much time to look into the headers to figure out if there is a trick to that but the `download` attribute seems to be a good solution to this problem.
